sphinx_rtd_theme/footer.html

```diff
@@ -49,8 +50,8 @@
     {% set readthedocs_web = '<a href="https://readthedocs.org">Read the Docs</a>'  %}
       {% trans sphinx_web=sphinx_web, readthedocs_web=readthedocs_web %}Built with {{ sphinx_web }} using a{% endtrans %} <a href="https://github.com/rtfd/sphinx_rtd_theme">{% trans %}theme{% endtrans %}</a> {% trans %}provided by {{ readthedocs_web }}{% endtrans %}.
   {%- endif %}
+  {%- endblock %}
```

Comment:
This endblock should be moved. The start begins right after the div start, but the div ends on line 46. Anyone using this block would override only the closing tag of the div